main/options/options.cc

```diff
@@ -611,10 +611,14 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
                                  "Extra parent directories which contain package files. These paths use an underscore "
                                  "package-munging convention, i.e. 'Project_Foo'",
                                  cxxopts::value<vector<string>>(), "string");
-    options.add_options(section)("extra-package-files-directory-prefix-slash",
+    options.add_options(section)("extra-package-files-directory-prefix-slash-deprecated",
                                  "Extra parent directories which contain package files. These paths use a slash "
                                  "package-munging convention, i.e. 'project/foo'",
                                  cxxopts::value<vector<string>>(), "string");
+    options.add_options(section)("extra-package-files-directory-prefix-slash",
+                                 "Extra parent directories which contain package files. These paths are constant names "
+                                 "joined by slashes, i.e. 'Project/Foo'",
+                                 cxxopts::value<vector<string>>(), "string");
```

Comment:
The `"string"` is how the argument is named.

```
      --stripe-packages         Enable support for Stripe's internal Ruby package system
      --stripe-packages-hint-message arg
                                Optional hint message to add to packaging related errors
                                (default: "")
      --extra-package-files-directory-prefix-underscore string
                                Extra parent directories which contain package files.
                                These paths use an underscore package-munging convention,
                                i.e. 'Project_Foo'
      --extra-package-files-directory-prefix-slash string
                                Extra parent directories which contain package files.
                                These paths use a slash package-munging convention, i.e.
                                'project/foo'
      --allow-relaxed-packager-checks-for string
                                Packages which are allowed to ignore the restrictions set
                                by `visible_to` and `export` directives
      --package-skip-rbi-export-enforcement string
                                Constants defined in RBIs in these directories (and no
                                others) can be exported
```

I've been trying to align on `<...>` for the name when the argument is not meant to be literally `--foo=string`. Also maybe a more descriptive name?

```suggestion
                                 cxxopts::value<vector<string>>(), "<path>");
```

If you want to do this in a follow up change (and align all the options with this style), that's fine too.